website/content/en/preview/upgrade-guide/_index.md

```diff
@@ -36,38 +38,39 @@ When there is a breaking change we will:
 
 ## How Do We Find Incompatibilities
 
 Besides the peer review process for all changes to the code base we also do the followings in order to find
 incompatibilities:
-* (To be implemented) Automated tests to see if installing, uninstalling, upgrading from an older version and downgrading
-to an older version to check the compatibility of the application
-* (To be implemented) Turn the commands in our
-documentation into scripts that we can automatically run to find incompatibilities between the application and
-documentation
+* (To be implemented) Automated tests for installing, uninstalling, upgrading from an older version, and downgrading to an older version to check the compatibility of the application
+* (To be implemented) Turn the commands in our documentation into scripts that we can automatically run to find incompatibilities between the application and documentation
 
 ## Nightly Builds
 
-(To be implemented) Every night we will build and release everything that has been checked into the source code. This
-allows us to detect problems including breaking changes and potential drifts in our external dependencies sooner than we
-otherwise would. It also allows some of advanced users who have their own nightly builds to test the upcoming changes
-before they are released.
+(To be implemented) Every night we will build and release everything that has been checked into the source code.
+This allows us to detect problems including breaking changes and potential drifts in our external dependencies sooner than we otherwise would.
+It also allows some of advanced users who have their own nightly builds to test the upcoming changes before they are released.
 
 ## Release Candidates
 
-(To be implemented) We are considering having release candidates when we are in major version 1. By adopting this
-practice we allow our users who are early adopters to test out new releases before they are available to the wider
-community thereby providing us with early feedback which would result in more stable releases.
+(To be implemented) We are considering having release candidates when we are in major version 1.
+By adopting this practice we allow our users who are early adopters to test out new releases before they are available to the wider community thereby providing us with early feedback which would result in more stable releases.
 
 ## Security Patches
 
-While we are in major version 0 we will not release security patches to older versions, rather we provide the patches in
-the latest versions. When at major version 1 we will have an EOL (end of life) policy where we provide security patches
+While we are in major version 0 we will not release security patches to older versions.
+Rather we provide the patches in the latest versions.
+When at major version 1 we will have an EOL (end of life) policy where we provide security patches
 for a subset of older versions and deprecate the others.
 
-# Released Upgrade Notes
```

Comment:
Can you undo removing this and "There are no published upgrade notes yet."?

This is meant to be a section where we will link to upgrade notes about incompatibility when they become available.  This was stated earlier in the doc as "Add a permanent separate file named `migrating_to_vx.y.z.md` to our website (linked at the bottom of this page)"

Reply:
I added those lines back in.